test/extended/testdata/bindata.go

```diff
@@ -43319,6 +43319,8 @@ func testExtendedTestdataImagestreamJenkinsSlavePodsYaml() (*asset, error) {
 var _testExtendedTestdataImagestreamtagJenkinsSlavePodsYaml = []byte(`apiVersion: v1
 kind: ImageStream
 metadata:
+  labels:
+    role: jenkins-slave
```

Comment:
Does the role have to be `jenkins-slave` - can it be `jenkins-agent`?

We should discourage the use of "slave" wherever possible, as it does not conform to our inclusive language standards (or those of the Jenkins community).

Reply:
yeah thses tests came into existence, are legacy conventions, before those standards were put into place.

a little more wide spread and non-trivial of a change than you think, but I'll start on it right now

Reply:
actually we are going to have to stage this @adambkaplan 

1) the sync plugin actually looks for both the label key `role` and the specific value `jenkins-slave`, and only that right now
2) so this change without a sync plugin change fails the test
3) and moving forward, while we can start looking for `role: jenkins-agent` we cannot break existing users still looking at the old value

So the path forward will be 
1) merge this as is
2) I'll update my sync plugin PR to look for either value
3) when that is in the image (i.e. 1.0.48 is what we pull in), we can go back and changes these tests

Reply:
> yeah thses tests came into existence, are legacy conventions, before those standards were put into place.

My main concern was that we were introducing a new labeling requirement that was not inclusive. If this was already in place, then we will need to watch for the old value and the new one.